File: beginner/javascript/1117.js
const input = require('fs').readFileSync('stdin', 'utf8');

const valores = input.split('\n');

let n1, n2, media, nota_valida = 0;

while (nota_valida !== 2) {
    const nota = parseFloat(valores.shift()); 

    if (nota < 0.0 || nota > 10.0) {
        console.log("nota invalida");
    } else {
        nota_valida++;
        if (nota_valida == 1) {
            n1 = nota;
        } else if(nota_valida == 2){
            n2 = nota;
        }

    }
}

media = (n1 + n2) / 2.0;

console.log(`media = ${media.toFixed(2)}`);

File: beginner/python/1117.py
n1 = 0
n2 = 0
media = 0
nota_valida = 0

while (nota_valida != 1):
    n1 = float(input())
    if(n1 < 0.0 or n1 > 10.0):
        print("nota invalida")
    else:
        nota_valida+=1


nota_valida = 0

while (nota_valida != 1):
    n2 = float(input())
    if(n2 < 0.0 or n2 > 10.0):
        print("nota invalida")
    else:
        nota_valida+=1


media = (n1 + n2) / 2.0

print("media = %.2f" % media)